### PR TITLE
Fixed the issue - #31

### PR DESCRIPTION
**Pull request by ayush-sleeping**
Hey @rajivharlalka ,
Fixed issue - Impressive & Informative Project Readme #31

Checkout the changes I have made and if you want to me to add more things in that, Let me know ..................
and if everything is good to go , Kindly merge it .

You can see the Output here, Checkout here
https://github.com/ayush-sleeping/Accio

### CODE REVIEW

**Comment by rajivharlalka** [comment]
Thanks @ayush-sleeping for the PR, the Readme looks nice now. Just a small change that would be good. in the `Open Source programs the repo has been a part of ` instead of Kharagpur Open Source Society it Would be Kharagpur Winter of Code and it's logo. The link is fine as it redirects to kwoc's website.

File: readme.md [hunk 1]
```diff
@@ -1,32 +1,138 @@
...
+## Open Source programs this repo has been part of
+<a href="https://github.com/Rayman-Sodhi/Clone-IT"><img src="https://badges.frapsoft.com/os/v1/open-source.svg?v=103"></a>
+
+<div align="center">
+<img src="https://media-exp1.licdn.com/dms/image/C560BAQGNflW7cZMukg/company-logo_200_200/0/1589034084843?e=1646870400&v=beta&t=mKip_R8iZ5ph54NuyudJ2dmcdp9hSuespgF4exp1ZQ0" width="150px">
+</div>
+
+<div align="center">
+    <a href="https://kwoc.kossiitkgp.org/">Kharagpur Winter of Code</a>
+
```
Comment (ayush-sleeping) [inline readme.md:88]: Checkout this, Changed the required things ........

Comment (rajivharlalka) [reply readme.md:88]: Thanks for the quick changes, looks much better

**Comment by rajivharlalka** [comment]
LGTM , any suggestions from your side @chirag828049 ?

**Comment by chirag828049** [comment]
Awesome work.
Just a little nitpicking. The spelling of "repository" is wrong. 
Everything else great. Fix that, I'll merge the PR.
Thanks.

**Comment by ayush-sleeping** [comment]
> Awesome work. Just a little nitpicking. The spelling of "repository" is wrong. Everything else great. Fix that, I'll merge the PR. Thanks.

thanks for noticing, I fixed the part . Checkout ......

**Comment by chirag828049** [comment]
Great!

**Pull request merged**